src/data/resolvers/motions.ts

```diff
@@ -0,0 +1,110 @@
+import { Resolvers } from '@apollo/client';
+
+import {
+  ClientType,
+  getLogs,
+  getBlockTime,
+  ExtensionClient,
+} from '@colony/colony-js';
+
+import { Context } from '~context/index';
+import { ColonyAndExtensionsEvents } from '~types/index';
+import {
+  ActionsPageFeedType,
+  SystemMessage,
+  SystemMessagesName,
+} from '~dashboard/ActionsPageFeed';
+import { NetworkMotionState } from '~utils/colonyMotions';
+
+import { ProcessedEvent } from './colonyActions';
+
+export const motionsResolvers = ({
+  colonyManager: { networkClient },
+  colonyManager,
+}: Required<Context>): Resolvers => ({
+  Query: {
+    async motionsSystemMessages(_, { motionId, colonyAddress }) {
+      const { provider } = networkClient;
+      const votingReputationClient = (await colonyManager.getClient(
+        ClientType.VotingReputationClient,
+        colonyAddress,
+      )) as ExtensionClient;
+      const motion = await votingReputationClient.getMotion(motionId);
+      const motionNetworkState = await votingReputationClient.getMotionState(
+        motionId,
+      );
+      const systemMessages: SystemMessage[] = [];
+
+      // @ts-ignore
+      const motionStakedFilter = votingReputationClient.filters.MotionStaked(
+        motionId,
+        null,
+        null,
+        null,
+      );
+      const motionStakedLogs = await getLogs(
+        votingReputationClient,
+        motionStakedFilter,
+      );
+      // @ts-ignore
+      // eslint-disable-next-line max-len
```

Comment:
Please leave some comments so that in the future we'll remember why these were ignored 

_(because of `colonyJS` obviously, but we might forget :P)_